bin/eslint.js

```diff
@@ -4,4 +4,8 @@ const CODE_DIR = "/code";
 process.chdir(CODE_DIR);
 
 const ESLint = require("../lib/eslint");
-ESLint.run({ dir: CODE_DIR });
+const issues = ESLint.run({ dir: CODE_DIR });
```

Comment:
I think this is a good solution: https://github.com/codeclimate/codeclimate-eslint/pull/230